src/webui/www/private/scripts/contextmenu.js

```diff
@@ -61,9 +61,9 @@ window.qBittorrent.ContextMenu = (function() {
                 x: 0,
                 y: 0
             },
-            onShow: $empty,
-            onHide: $empty,
-            onClick: $empty,
+            onShow: () => {},
```

Comment:
This code is incomprehensible to me. You replaced some variables with empty lambdas. Could you explain? Can't we just delete the relevant fields (onShow, onHide, onClick)?

Reply:
>You replaced some variables with empty lambdas. Could you explain?

Those 'variables' are in fact empty functions. See: https://mootools.net/core/docs/1.6.0/Core/Core#Deprecated-Functions:empty
And I used arrow function expression since it is shorter than `function() {}`

>Can't we just delete the relevant fields (onShow, onHide, onClick)?

In theory sub classes should be able to do it without the base class definition but I suppose the original author wants it to be clear.
I think it is better to keep it.

Reply:
In C++ it would look like the following:
```c++
void Subclass::onClick() // override
{
    Baseclass::onClick();
}
```
...which is really redundant.

However, I leave JS at your discretion.